docs/source/howto/calculations/pw.rst

```diff
@@ -311,3 +311,38 @@ To retrieve additional output files, specify the list of files in the ``ADDITION
 
 .. |SSSP| replace:: Standard Solid-State Pseudopotentials (SSSP)
 .. _SSSP: https://www.materialscloud.org/discover/sssp/table/efficiency
+
+
+How to analyze the results
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a ``PwCalculation`` is completed, there are quite a few possible analyses to perform.
```

Comment:
This is not incorrect, but it is sort of funny that we say this and then only show one. 😉

Reply:
Sure, but I imagine we will be adding different analyses in the future. This is the first one to kick things off.